projects/packages/videopress/src/class-wpcom-rest-api-v2-endpoint-videopress.php

```diff
@@ -147,6 +147,21 @@ public function register_routes() {
 			)
 		);
 
+		// Endpoint to know if the video metadata is editable.
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/(?P<video_guid>\w+)/belong-to/(?P<post_id>\d+)/',
```

Comment:
`belongs-to`

Reply:
Thanks for the suggestion. Addressed here https://github.com/Automattic/jetpack/pull/30427/commits/51517b0b9600dbd042f1a3315e6bcebe929e03fa